### PR TITLE
Add rosé pine dawn and moon themes

### PR DESCRIPTION
**Pull request by JaninaWibker**
This adds the dawn and moon themes (both source files and minified files) and updates the readme accordingly.

I updated `rosepine.css` to use css variables. The 2 new files are essentially just copies of it with different variables.

I'm not sure whether or not including screenshots for the readme.md file in the repo is what you want, if you'd like to I can update it.

### CODE REVIEW

**Comment by mvllow** [comment]
Thank you for the new variants! If you wouldn't mind, please do remove the screenshots from the repo and use the GitHub cdn. Drag an image into any textarea on GitHub, e.g. in these comments, for a link.

Other than that, I would like to rename rosepine in the filenames to rose-pine but I am happy to do that in a follow-up PR because what you did matches what we have currently :)

**Comment by JaninaWibker** [comment]
I can update the filenames and remove the assets folder, no problem!

One concern I have is that renaming `rosepine.min.css` would break other peoples code as the (previous) link in the readme file wouldn't work anymore.
Should I rename all files including the files in the `min/` folder or just the files in the base directory?

**Comment by mvllow** [comment]
Yea, I was afraid of that – honestly I wouldn't recommend anyone to use that link for a production site as it's too fragile. This should either be removed (in my opinion) or a warning placed above to not use that link in production. Again, I don't mind making any of these changes either but I do appreciate all the work you're putting into this :)

I know this isn't the best indicator of who is using this but [searching "rosepine.min.css"](https://github.com/search?q=rosepine.min.css&type=code) throughout GitHub only comes with relative links, none pointing to this repo.

**Comment by JaninaWibker** [comment]
I renamed the files in the `min/` directory as well and added a small note to the readme file about possible changes to the links

File: readme.md [hunk 1]
```diff
@@ -16,17 +16,48 @@
...
+/>
+```
+
+> Note: these links might stop working at some point due to renames or similar repository changes
+
+
 ## Gallery
 
-### As seen on https://stepbro.voring.me
+### Rosé Pine
```
Comment (mvllow) [inline readme.md]:
```suggestion
**Rosé Pine**
```

File: readme.md [hunk 2]
```diff
@@ -16,17 +16,48 @@
...
+
+![rose-pine](https://user-images.githubusercontent.com/16372321/227316298-6668954d-859e-439d-8a1d-2fb79346c100.jpg)
```
Comment (mvllow) [inline readme.md]:
```suggestion
![highlight.js with Rosé Pine](https://user-images.githubusercontent.com/16372321/227316298-6668954d-859e-439d-8a1d-2fb79346c100.jpg)
```

File: readme.md [hunk 3]
```diff
@@ -16,17 +16,48 @@
...
+
+
+### Rosé Pine Moon
```
Comment (mvllow) [inline readme.md]:
```suggestion
**Rosé Pine Moon**
```

File: readme.md [hunk 4]
```diff
@@ -16,17 +16,48 @@
...
+
+![rose-pine-moon](https://user-images.githubusercontent.com/16372321/227316293-579c772a-ab7b-4b8f-ae19-4bafc61b0c99.jpg)
```
Comment (mvllow) [inline readme.md]:
```suggestion
![highlight.js with Rosé Pine Moon](https://user-images.githubusercontent.com/16372321/227316293-579c772a-ab7b-4b8f-ae19-4bafc61b0c99.jpg)
```

File: readme.md [hunk 5]
```diff
@@ -16,17 +16,48 @@
...
+
+
+### Rosé Pine Dawn
```
Comment (mvllow) [inline readme.md]:
```suggestion
**Rosé Pine Dawn**
```

File: readme.md [hunk 6]
```diff
@@ -16,17 +16,48 @@
...
 
-![image](https://user-images.githubusercontent.com/44733677/156076709-f01a5624-d50a-4e09-9bae-48494802ef07.png)
+![rose-pine-dawn](https://user-images.githubusercontent.com/16372321/227316267-5d98c0ed-2046-47f1-98a6-213588297abd.jpg)
```
Comment (mvllow) [inline readme.md]:
```suggestion
![highlight.js with Rosé Pine Dawn](https://user-images.githubusercontent.com/16372321/227316267-5d98c0ed-2046-47f1-98a6-213588297abd.jpg)
```

**Review by mvllow — CHANGES_REQUESTED** [review]
A few nitpicks and this should be good to go :)

**Comment by mvllow** [comment]
Thanks again for all your time/work put into this! Lovely PR 💜

**Pull request merged**